crates/chronicle-domain-test/src/test.rs

```diff
@@ -127,7 +131,63 @@ mod test {
         let keystore = DirectoryStoredKeys::new(keystore_path).unwrap();
         keystore.generate_chronicle().unwrap();
 
-        let tp = EmbeddedChronicleTp::new();
+        let mut buf = vec![];
+        Setting {
+            entries: vec![Setting_Entry {
+                key: "chronicle.opa.policy_name".to_string(),
```

Comment:
Where we're reusing a lot of these strings I wonder if it's worth defining them as constants somewhere (perhaps in some settings enum).